GAD 6 - Verbessertes Sortieren/tests/simplesort/TesterDualQuicksortResult.java

```diff
@@ -1,16 +1,23 @@
 package tests.simplesort;
 
+import gad.simplesort.DualPivotFinder;
 import gad.simplesort.Result;
 
 import java.util.Arrays;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class TesterDualQuicksortResult implements Result {
-    int[] startResult = null;
+    int[] numbersAtStart = null;
     int[] indicesOfLog = new int[6];
     int countThroughLog = 0;
 
+    public TesterDualQuicksortResult(DualPivotFinder pivotFinder) {
+        this.pivotFinder = pivotFinder;
+    }
+
+    DualPivotFinder pivotFinder;
```

Comment:
Could you please move this above the constructor for improved readability? That way all attributes are grouped together.